### PR TITLE
[STK-50] Add CI workflow

### PR DESCRIPTION
**Pull request by wixzi**

### CODE REVIEW

**Comment by wixzi** [comment]
> Can you update .gitignore so there aren't build files on our repo?

`The cache folder is by default stored within your project folder (in .yarn/cache). Just make sure you add it to your repository`

I was trying this.
https://yarnpkg.com/features/zero-installs

File: packages/app/app/layout.tsx [hunk 1]
```diff
@@ -14,6 +14,33 @@ interface RootLayoutProps {
...
+        />
+        <link rel="manifest" href="/assets/favicons/site.webmanifest" />
+        <link
+          rel="mask-icon"
+          href="/assets/favicons/safari-pinned-tab.svg"
+          color="#F8F8F7"
+        />
+        <meta name="msapplication-TileColor" content="#F8F8F7" />
+        <meta name="theme-color" content="#A2E771" />
+      </head>
```
Comment (Diogomartf) [inline packages/app/app/layout.tsx:43]: I commented in the header PR, we don't need to set icons like this. Just put the favicon.ico on the root folder.

**Comment by berteotti** [comment]
Don't forget to update this branch with the base branch

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - master
+      - develop
```
[the remark this answers is not in the document]
Comment (wixzi) [reply .github/workflows/CI.yml]: Added `production` branch

**Review by berteotti — APPROVED** [review]
Good job Velu!

**Review by Diogomartf — APPROVED** [review]

**Pull request merged**